articles/introduction-to-sequalize-orm-for-nodejs/index.md

```diff
@@ -0,0 +1,215 @@
+### Introduction to Sequelize ORM for Node.js
+![hero image](hero.jpg)
+
+### Introduction
+Connecting an Object-relational Mapper to a Node.js application is not an easy task for most developers since there are no better elaborate resources and documentation about it. If you're one, then this article is for you. It also covers SQL basics so beginners in databases will find it insightful. We will not dive though into how Sequelize ORM integrates with a Nodejs application. Let us save that for a future tutorial.
+
+|Table of Content           |
+|-------------------------  |
+|Prerequisites              |
+|SQL database basics        |
+|Object Relational Mappers  |
+|Sequelize                  |
+|Benefits of Sequelize      |
+|Conclusion                 |
```

Comment:
```suggestion
|Table of Content           |
|-------------------------  |
|[Prerequisites](#prerequisites)              |
|[SQL database basics](#sql-database-basics)        |
|[Object Relational Mappers](#object-relational-mappersorm-nodejs-approaches)  |
|[Sequelize](#what-is-sequelize)                  |
|[Benefits of Sequelize](#benefits-of-sequelize)      |
|[Conclusion](#conclusion)                 |
```
This looks good. You need not put them as a table, a simple one would be enough. Anyway it's fine.

Please note this change. I have linked the markdown headers to table of contents. By doing this, the reader will be able to jump to the respective header easily, without scrolling.